python-mini-projects/projects/convert pdf to text/converter1.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+
+
+import PyPDF2
+import os
+
+
+if(os.path.isdir("temp") == False):
+    os.mkdir("temp")
+    
+txtpath = ""
+pdfpath = ""
+
+
+
+pdfpath = input("Enter the name of your pdf file - please use backslash when typing in directory path: ")   #Provide the path for your pdf here
+txtpath = input("Enter the name of your txt file - please use backslash when typing in directory path: ")   #Provide the path for the output text file  
+
+BASEDIR = os.path.realpath("temp") # This is the sample base directory where all your text files will be stored if you do not give a specific path
+print(BASEDIR)
+
+
+if(len(txtpath) == 0):
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To check if a container or sequence (string, list, tuple) is empty, use `if not val`. Do not compare its length using `if len(val) == 0`.

 [Learn more](https://www.python.org/dev/peps/pep-0008/#programming-recommendations#:~:text=if%20not%20seq)